src/ucp/core/ucp_ep.c

```diff
@@ -1167,6 +1167,8 @@ static void ucp_ep_params_check_err_handling(ucp_ep_h ep,
         return;
     }
 
+    ucs_assert(!ep->worker->context->config.ext.gva_enable);
```

Comment:
why needed?

Reply:
WDYT?

Reply:
AFAIR we agreed to introduce err_handling+gva in upcoming PR, meanwhile it's not supported

Reply:
but don't we already check it before reaching this point?

Reply:
yes